docs/usage/cli.rst

```diff
@@ -38,21 +43,21 @@ The autodiscovery follows these lookup locations in order:
 
 Within these locations, Litestar CLI looks for:
 
-1. An object named ``app`` that is an instance of :class:`Litestar <.app.Litestar>`
-2. An object named ``application`` that is an instance of :class:`Litestar <.app.Litestar>`
-3. Any object that is an instance of :class:`Litestar <.app.Litestar>`
-4. A callable named ``create_app``
-5. A callable annotated to return an instance of :class:`Litestar <.app.Litestar>`
+1. An :term:`object` named ``app`` that is an instance of :class:`Litestar <.app.Litestar>`
+2. An :term:`object` named ``application`` that is an instance of :class:`Litestar <.app.Litestar>`
+3. Any :term:`object` that is an instance of :class:`Litestar <.app.Litestar>`
+4. A :term:`callable` named ``create_app``
+5. A :term:`callable` annotated to return an instance of :class:`Litestar <.app.Litestar>`
 
 Commands
 --------
```

Comment:
Okay hear me out - We have some unique things in here but with a big note to maintainability and scalability, they are not terribly useful compared to what the Click directive gives use (see https://docs.litestar.dev/latest/reference/cli.html)

@litestar-org/maintainers wdyt about removing this and just moving the reference of the commands up some.

My only concern is things like https://docs.litestar.dev/latest/usage/cli.html#ssl.

I was thinking we could enhance the `help=` for those commands but that file would become garagantuan.. I'm not sure of a better way. We could have a few sections I guess for special things like this - but I feel like they would become stale without us seeing it. The "Help" for those seems already good enough.. e.g., `litestar run --help` -> look at what the subcommands are.

Thoughts?

Reply:
This too coupled with this massive ass table.

We could move it to a csv or similar sphinx table structure but I don't think it solves my concerns.

Reply:
No pressure on just keeping it, too.